app/src/main/java/com/jerboa/ui/components/remove/comment/CommentRemoveActivity.kt

```diff
@@ -0,0 +1,95 @@
+package com.jerboa.ui.components.remove.comment
+
+import android.util.Log
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.outlined.Send
+import androidx.compose.material3.Scaffold
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.saveable.rememberSaveable
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalFocusManager
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.lifecycle.viewmodel.compose.viewModel
+import com.jerboa.JerboaAppState
+import com.jerboa.R
+import com.jerboa.api.ApiState
+import com.jerboa.db.entity.isAnon
+import com.jerboa.model.AccountViewModel
+import com.jerboa.model.RemoveViewModel
+import com.jerboa.ui.components.common.ActionTopBar
+import com.jerboa.ui.components.common.getCurrentAccount
+import com.jerboa.ui.components.remove.RemoveItemBody
+import it.vercruysse.lemmyapi.v0x19.datatypes.Comment
+
+object CommentRemoveReturn {
+    const val COMMENT_VIEW = "comment-remove::return(comment-view)"
+    const val COMMENT_SEND = "comment-remove::send(comment-view)"
```

Comment:
These seem redundant, since they serve the same purpose as `CommentEditReturn.COMMENT_XXX` anyway. I spose it makes sense to have them be explicit tho.